includes/wikia/models/NavigationModel.class.php

```diff
@@ -453,62 +420,62 @@ public function parseLines($lines, $maxChildrenAtLevel = array()) {
 					if ( $depth == $lastDepth + 1 ) {
 						$parentIndex = $i;
 					} else if ( $depth == $lastDepth ) {
-						$parentIndex = $nodes[$i][ self::PARENT_INDEX ];
+						$parentIndex = $nodes[$i][self::PARENT_INDEX];
 					} else {
-						for( $x = $i; $x >= 0; $x-- ) {
+						for ( $x = $i; $x >= 0; $x-- ) {
 							if ( $x == 0 ) {
 								$parentIndex = 0;
 								break;
 							}
-							if ( $nodes[$x][ self::DEPTH ] <= $depth - 1 ) {
+							if ( $nodes[$x][self::DEPTH] <= $depth - 1 ) {
 								$parentIndex = $x;
 								break;
 							}
 						}
 					}
 
-					if ( isset( $maxChildrenAtLevel[$depth-1] ) ) {
-						if ( isset( $nodes[ $parentIndex ][ self::CHILDREN ] ) ) {
-							if ( count( $nodes[ $parentIndex ][ self::CHILDREN ] ) >= $maxChildrenAtLevel[$depth-1] ) {
+					if ( isset( $maxChildrenAtLevel[$depth - 1] ) ) {
+						if ( isset( $nodes[$parentIndex][self::CHILDREN] ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)